src/pages/_document.tsx

```diff
@@ -47,4 +47,22 @@ export default class MyDocument extends Document {
       sheet.seal();
     }
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
```

Comment:
😏

Reply:
Not my fault if `Next.js` can't type correctly their APIs 👀.